### PR TITLE
Feature/v1 021/connection to API for getting profile info

### PR DESCRIPTION
**Pull request by Julia-Maliutsina**
1. Added getting access token from cookie.
2.  Passing it to function that requests profile info.
3.  Displaying data from response on Profile page.

### CODE REVIEW

File: frontend/src/api/profile/getProfile.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      lastName?: string;
+      skills?: Array<string>;
+      courses?: Array<string>;
+      avatar?: string;
+      birthday?: Date;
+      skype?: string;
+      error?: unknown;
+    };
+    try {
+      const response = await apiClient.get('/api/account/profile', {
```
Comment (k-krasilnikova) [inline frontend/src/api/profile/getProfile.ts:19]: Move please this route to the constants (create common collection for them)

File: frontend/src/api/profile/getProfile.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        headers: {
+          Authorization: `Bearer ${accessToken}`,
+        },
```
Comment (k-krasilnikova) [inline frontend/src/api/profile/getProfile.ts:22]: Let's describe auth header inside `apiClient`. It's not a good idea to define it each time for each route. Just check is client authorized(at this moment always true) inside API client wrapper.

File: frontend/src/pages/Profile/ProfileContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import useGetProfile from 'api/profile';
+import accessTokenGetter from 'utils';
+
+import Profile from './Profile';
+
+const ProfileContainer: React.FC = () => {
+  const accessToken = accessTokenGetter();
```
Comment (k-krasilnikova) [inline frontend/src/pages/Profile/ProfileContainer.tsx:9]: Remove it please. It should be inside `apiClient`

File: frontend/src/utils/getAccessToken.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+const accessTokenGetter = () => {
+  const cookies = document.cookie.split(';');
+  for (let cookie = 0; cookie < cookies.length; cookie++) {
+    if (/accessToken=/.test(cookies[cookie])) {
```
Comment (k-krasilnikova) [inline frontend/src/utils/getAccessToken.ts:4]: Into constant!

File: frontend/src/utils/getAccessToken.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+      const accessToken = cookies[cookie].slice(cookies[cookie].indexOf('=') + 1);
+      return accessToken;
```
Comment (k-krasilnikova) [inline frontend/src/utils/getAccessToken.ts:6]: Maybe it will be better to use some library instead of describing bike from zero?

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]

File: frontend/src/utils/getAccessToken.ts (hunk 5 above)
Comment (Julia-Maliutsina) [reply frontend/src/utils/getAccessToken.ts:6]: used library js-cookie

**Review by hauteballe — APPROVED** [review]

**Review by k-krasilnikova — APPROVED** [review]

**Pull request merged**